rllib/utils/test_utils.py

```diff
@@ -1215,9 +1215,20 @@ def run_rllib_example_script_experiment(
         algo = config.build()
         for iter in range(args.stop_iters):
             results = algo.train()
-            print(f"R={results['episode_reward_mean']}")
+            print(f"R={results['sampler_results']['episode_reward_mean']}", end="")
+            if "evaluation" in results:
+                Reval = results["evaluation"]["sampler_results"]["episode_reward_mean"]
```

Comment:
Somewhere in the docs we should give a clear structure of this dictionary (`results`) in general, such that users understand where to get certain results and where to put custom ones. 

 This can go together with the `reduce_fn`  in the learner (we want to clean this up and make it more user-friendly)